tests/test_jwt.py

```diff
@@ -137,8 +137,10 @@ def test_verify_id_token_with_certs_uri_default_http(self):
         jwt = self._create_signed_jwt()
 
         http = http_mock.HttpMock(data=datafile('certs.json'))
+        patch = mock.patch(
+            'oauth2client.transport.get_cached_http', return_value=http)
 
-        with mock.patch('oauth2client.transport._CACHED_HTTP', new=http):
```

Comment:
I'm not sure what you're saying. The mock isn't gone, it's been replaced with the patch above.